### PR TITLE
let plugin package install and test on CentOS

### PR DESCRIPTION
**Pull request by d-w-moore**
add CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION to avoid package file conflicts with iRODS packages and allow centos7 tests to pass for 4.3.0 release

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -95,6 +95,16 @@ foreach(IRODS_CURRENT_PLUGIN_TARGET ${IRODS_CURL_PLUGINS})
...
 
+if (NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/" CACHE STRING "Package root path. \"/\" is correct for normal package builds.." FORCE)
+  message(STATUS "Setting unspecified CPACK_PACKAGING_INSTALL_PREFIX to '${CPACK_PACKAGING_INSTALL_PREFIX}'. This is the correct setting for normal builds.")
+endif()
+
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts/irods")
+list(APPEND CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION "${CPACK_PACKAGING_INSTALL_PREFIX}var/lib/irods/scripts/irods/test")
```
Comment (SwooshyCueb) [inline CMakeLists.txt:106]: Please follow the pattern from the python repo:
https://github.com/irods/irods_rule_engine_plugin_python/blob/8f7356e499e60a4e55475506b0078a02d115f842/CMakeLists.txt#L128-L131

**Review by SwooshyCueb — CHANGES_REQUESTED** [review]

File: CMakeLists.txt [hunk 2]
```diff
@@ -135,6 +135,16 @@ endif()
 
 include(IrodsCPackCommon)
 
+if (NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/" CACHE STRING "Package root path. \"/\" is correct for normal package builds.." FORCE)
+  message(STATUS "Setting unspecified CPACK_PACKAGING_INSTALL_PREFIX to '${CPACK_PACKAGING_INSTALL_PREFIX}'. This is the correct setting for normal builds.")
+endif()
```
[the remark this answers is not in the document]
Comment (SwooshyCueb) [reply CMakeLists.txt:141]: I'm fairly certain this isn't needed either way. Try a clean build without this bit and see if the rpms come out okay

[the remark this answers is not in the document]
Comment (d-w-moore) [reply CMakeLists.txt:141]: Will do.

**Comment by d-w-moore** [comment]
squashed/#'d

**Review by alanking — APPROVED** [review]

**Pull request merged**